scripts/apps/desks/views/desk-select.html

```diff
@@ -10,7 +10,7 @@
             <button ng-click="onChange({desk: desk})" sd-desk-select-item>{{ :: desk.name }}</button>
         </li>
         <li ng-if="sendToPersonal">
-            <button ng-click="onChange({desk: {'_id': 'Personal Space'}})">Personal Space</button>
+            <button ng-click="onChange({desk: personalSpace})" translate>PERSONAL SPACE</button>
```

Comment:
It'd be good to use CSS for upper-casing so we generate less strings for translation.